go/api/response_handlers.go

```diff
@@ -143,13 +143,13 @@ func parseSet(response *C.struct_CommandResponse) (interface{}, error) {
 		return nil, nil
 	}
 
-	slice := make(map[interface{}]struct{}, response.sets_value_len)
+	slice := make(map[string]struct{}, response.sets_value_len)
```

Comment:
Same here, are we certain that we won't have any other use case for `Set`? Do we not have any use case for `Set of int64` or `Set of flota64`?

Reply:
Same with sets